source/Octopus.Tentacle.Tests/Octopus.Tentacle.Tests.csproj

```diff
@@ -29,7 +29,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
     <ProjectReference Include="..\Octopus.Tentacle.CommonTestUtils\Octopus.Tentacle.CommonTestUtils.csproj" />
-    <ProjectReference Include="..\Octopus.Tentacle\Octopus.Tentacle.csproj" />
```

Comment:
The Test project doesn't depend on the app project?! What am I missing here?

Reply:
Inherited dependencies